packages/manager/modules/dbaas-logs/src/logs/translations/Messages_fr_FR.json

```diff
@@ -152,6 +152,40 @@
   "logs_home_account_name": "Nom",
   "logs_home_tokens": "Jetons d'API",
   "logs_home_tokens_tooltip": "Ces jetons (tokens) vous permettent d'accéder aux APIs",
+  "logs_encryption_keys_title": "Clés de chiffrements PGP",
+  "logs_home_encryption_keys_tooltip": "Ces clés de chiffrements PGP vous permettent de chiffrer les archives produites par vos flux de données",
+  "logs_encryption_keys_description": "Vous pouvez associer ces clés de chiffrements PGP à vos flux de donnés sur lesquels la rétention à long terme est activée. Chaque archive produite pour ce flux de donnés sera alors chiffrée avec la/les clé(s) de chiffrement PGP qui y sont assignées.",
+  "logs_encryption_keys_goto_1": "Gérer mes",
+  "logs_encryption_keys_goto_2": "clés de chiffrements PGP",
+  "logs_encryption_keys_add": "Ajouter une clé de chiffrement PGP",
+  "logs_encryption_keys_col_title": "Nom",
+  "logs_encryption_keys_col_fingerprint": "Fingerprint",
+  "logs_encryption_keys_col_algorithm": "Algorithme",
+  "logs_encryption_keys_col_uid": "UID",
+  "logs_encryption_keys_col_nb_archives": "Nombre d'archives",
+  "logs_encryption_keys_col_nb_archives_full": "Nombre d'archives chiffrées avec cette clé PGP",
+  "logs_encryption_keys_col_content": "Contenu",
+  "logs_encryption_keys_col_createdAt": "Date de création",
+  "logs_encryption_keys_add_description": "Ajouter ici les informations de votre clé de chiffrement PGP.",
+  "logs_encryption_keys_add_title_tooltip": "Nom d'affichage de votre clé de chiffrement PGP au sein d'OVHcloud",
+  "logs_encryption_keys_add_title_placeholder": "My work laptop PGP key",
+  "logs_encryption_keys_add_fingerprint_tooltip": "Fingerprint de votre clé PGP",
+  "logs_encryption_keys_add_fingerprint_placeholder": "46DA7E800B7D9CF8A0A1AD1557353365699DDF39",
+  "logs_encryption_keys_add_content_tooltip": "Contenu de votre clé PGP, au format ASCII armored",
+  "logs_encryption_keys_add_content_placeholder": "-----BEGIN PGP PUBLIC KEY BLOCK-----\n\nmDMEYxiLtxYJKwYBBAHaRw8BAQdAy4ggQveZ1367awSh4HdpeYokD4Rs16hfqUTU\nK+NZYYW0H1JvYmVydG8gQ2FybG9zIDxyb2JlcnRAY2FybC5vcz6IkAQTFggAOBYh\nBEbafoALfZz4oKGtFVc1M2Vpnd85BQJjGIu3AhsDBQsJCAcCBhUKCQgLAgQWAgMB\nAh4BAheAAAoJEFc1M2Vpnd85CV4BAMclcRA6QfiHrtgGRiOgx1XuqSfV3245VtDl\nhqkiuyJWAP0XBTkgqUmQ3QSoE+fYOpnWpHt6n1m7FikkadOrrOmaCLg4BGMYi7cS\nCisGAQQBl1UBBQEBB0Ct7vcRp8DniPIvkMREAikuZBlWnJv8TyzSK4JtIcCXIwMB\nCAeIeAQYFggAIBYhBEbafoALfZz4oKGtFVc1M2Vpnd85BQJjGIu3AhsMAAoJEFc1\nM2Vpnd85ZnYA/jQ9zyXG2WYT6OoaDdk2/g4uIA5qaF0VZM207GFpCFuQAQDrJFhQ\ngzZPPx4UmQYPpVJO+uH1ZxKWIzZfVZAFZJw+Bg==\n=Q81L\n\n-----END PGP PUBLIC KEY BLOCK-----",
```

Comment:
**nitpick:** Please maintain this as a constant since this cannot be translated.

Reply:
Fixed, moved to `Logs.Constants` file where all our constants are (for the 3 placeholders, as none of these 3 will need translations)